cmd/edgectl/aes_install_errors.go

```diff
@@ -93,103 +100,30 @@ func (i *Installer) GetVersionsError(err error) Result {
 	}
 }
 
-// Unable to fetch the AES CRD manifests (aes-crds.yaml)
-func (i *Installer) AESCRDManifestsError(err error) Result {
+// Unable to download the latest Chart
+func (i *Installer) DownloadError(err error) Result {
 	i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
 
 	url := "https://www.getambassador.io/docs/latest/topics/install/help/aes-crd-manifests"
```

Comment:
This needs to be a new help page (with a new URL).

Reply:
Ok, fixed